### ISSUE TITLE
Branch: Prepare an a11y demo for the strings team

### ISSUE THREAD

**Issue by AgustinVallejo** [phetsims/rosetta#464]

**Comment by AgustinVallejo** [comment]
During work on this I noticed something odd: A11y View doesn't work on ox-dev server because it's incorrectly adding the `_phet.html` to the filename, instead of just `_en.html`. Leaving it here for future discussion

**Comment by AgustinVallejo** [comment]
I've pushed an initial attempt that pretty much lays all the necessary pieces in place for the A11y Test to work. The biggest struggle is the following: As far as I understand, the sim's iframe needs a src to pull the sim from. In the servers it's obtained from the same folder (i.e. `greenhouse_a11y` uses `./greenhouse_en`) so it's fine. In our case, not only we don't have access to such file but what we've got is a raw text of the whole html with the strings replaced.

I tried setting a URI the iframe could load but all it shows is `about:blank#blocked`. I also managed to get it to show a sim by passing directly the sim's URL from the server, which is what's currently pushed. But that's no solution at all, and it also doesn't load the PDOM.
